tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "./tsconfig.base.json",
+    "compilerOptions": {
+      "outDir": "src/esm",
+      "esModuleInterop": true,
```

Comment:
esModuleInterop is only useful for commonjs

ESM does not need interop considerations with ESM.

Reply:
you're right..., will move it